career/general_personality.md

```diff
@@ -0,0 +1,93 @@
+
+### How do you handle bugs in your code?
+
+Debugging
+
+Creating code is only part of a programmer’s job. When software doesn’t work as expected, a programmer is expected to get to the root of the problem quickly and effectively. Instead of spending hours blindly making changes, search for a programmer who prefers to carefully investigate his code and research possible issues until an answer is found.
+
+### Describe your ideal work environment
+
+Work environment match
+
+Some programmers require complete silence to concentrate, while others thrive in chaos. A worker’s personal preferences are an important part of his productivity, so it’s best to be clear about your office environment up front to avoid problems after the person is hired.
+
+### What are your passion projects/hobbies?
+
+Passion for the work
+
+This is a tricky one. Many employers think that true programmers must be “computer geeks”, spending their time gaming, building servers or creating apps for friends. While this passion isn't a necessity, it's often a way to find top-shelf candidates. People that attend our bootcamp are not code obsessives with no other dimension to their personality. They do not devote every single moment of their life to coding and the don't live the stereotyped developer lifestyle. But we still argue that they are passionate - in a way that makes them, in a long run, more valuable to an organisation than people who code every second of every day.
+
+Bootcamp graduates are self-motivated, embrace challenge and are intelligent problem solvers with a broad set of skills and experience. They have the self-confidence to make an informed choice about pivoting into a new profession and planning for the long term. They are also tough - tough enough to make it through a bootcamp, an endeavor that not only requires them to learn a completely new skill-set but quite often reassesses their whole approach towards how they function as professionals.
+
+Being able to do all that takes a lot of courage, determination, motivation and .., yes, passion.
+
+### Describe a time when you were under extreme pressure and your application wasn’t working. What did you do?
+
+Grace under fire
+
+It may not seem like it to the outsider, but programming can be an extremely stressful profession. When deadlines are tight and nothing seems to be working, it's easy to freeze up. We argue that the ideal candidate should be able to handle even the most stressful situations calmly and, most importantly, be able to continue working. The challenge is to be able to showcase situations like that from your previous career or the camp itself and tell a compelling story about how you dealt with overcoming the obstacles.
```

Comment:
`It may not seem like it to the outsider,`  `It may not seem like it, to an outsider,`